### ISSUE TITLE
UX: Underneath View

### ISSUE THREAD

**Issue by DorisAtCeR** [UoA-eResearch/mt_ngauruhoe_3js#4]
Can you prevent the user from titling the whole mountain to the extend where they see it from underneath? That would be nice.

**Comment by neon-ninja** [comment]
https://github.com/UoA-eResearch/mt_ngauruhoe_3js/commit/bfdd9ed8e160ce5c23a34e4f01eebcbef1f3c04a is this enough?

**Comment by DorisAtCeR** [comment]
Maybe a tiny bit more, but it looks much better than before.

**Comment by neon-ninja** [comment]
https://github.com/UoA-eResearch/mt_ngauruhoe_3js/commit/99292c4b1c490b558aa026bab30c9ff48cf0fc9b is this enough?

**Comment by DorisAtCeR** [comment]
No, it works for pan and tilt but when you zoom, again, you can see underneath.

**Comment by neon-ninja** [comment]
How about this? https://github.com/UoA-eResearch/mt_ngauruhoe_3js/commit/aef5309774cd246d96b6e66bccd1c5b8ac0ed576

**Comment by DorisAtCeR** [comment]
![screen shot 2018-07-09 at 4 03 31 pm](https://user-images.githubusercontent.com/26263513/42430166-af6ce34c-8391-11e8-8942-c0636bca65e2.png)
 nope, not quite
